docs/packages.md

```diff
@@ -0,0 +1,121 @@
+# DPI Packages
+
+Ruter publishes a packages manifest that provides a list of packages that should be installed on vehicles at the following locations.
+
+| Environment | URL | Purpose |
+| --- | ---| --- |
+| Prod | https://bus-dpi.transhub.io/prod/packages.json | All vehicle running regular routes |
+| Stage | https://bus-dpi.transhub.io/stage/packages.json | Test rigs, vehicles being tested before running regular routes |
+
+These locations should be checked once daily, at a minimum, after 16.00 CET, which is our cutoff for changes for the day, Monday to Friday.
+
+It is expected that new packages should be installed on all vehicles running in regular traffic after 05.00 CET the following day.
+
+## Contents of Manifest
+
+```json
+{
+  "timestamp" : "2021-02-25T14:43:14Z",
+  "packages" : [ {
+    "destination" : "application",
+    "url" : "https://bus-dpi.transhub.io/packages/application-2021-02-25T14-25-12Z.zip",
+    "sha256" : "5258f1e757ab9fb1e222601c793030219b7064b21206d234fc094bbdaf546fd0"
+  }, {
+    "destination" : "application/media",
+    "url" : "https://bus-dpi.transhub.io/packages/media-2019-10-16T15-11-00Z.zip",
+    "sha256" : "44a143c13ab452b7bd5d06f4ce3373af7b5a4da0095b42a3db27bd04e172072b"
+  } ]
+}
+```
+
+When a new version of the packages is published the timestamp will increment.
+
+Use the URL to fetch the packages and check that the downloaded Zip file has the same SHA256 values as in the manifest.
+
+## Placement of Files
+
+Currently the two zip files are organized internally slightly differently, and this affects how they need to be handled. The destination in the manifest refers to where the working directory must be when the file is unzipped.
+
+In addition, the destination "application" should be thought of a location chosen by the operator, which we can call $APPLICATION_ROOT.
+
+The packages should end up deployed in a structure like this:
+
+* $APPLICATION_ROOT
+  * app
+  * media
+    
+The directory application can actually be anything as long as it is the root of the web site that is served.
+
+Manually setting up this structure would be done with the following steps (verified in an Alpine container in Docker):
+
+```sh
+~ # apk add curl tree
+~ # curl -O https://bus-dpi.transhub.io/packages/application-2021-02-25T14-25-12Z.zip
+~ # curl -O https://bus-dpi.transhub.io/packages/media-2019-10-16T15-11-00Z.zip
+~ # sha256sum application-2021-02-25T14-25-12Z.zip media-2019-10-16T15-11-00Z.zip
+5258f1e757ab9fb1e222601c793030219b7064b21206d234fc094bbdaf546fd0  application-2021-02-25T14-25-12Z.zip
+44a143c13ab452b7bd5d06f4ce3373af7b5a4da0095b42a3db27bd04e172072b  media-2019-10-16T15-11-00Z.zip
+~ # mkdir -P /var/www/application
+~ # cd /var/www/application
+/var/www/application # unzip ~/application-2021-02-25T14-25-12Z.zip
+/var/www/application # mkdir media
+/var/www/application # cd media
+/var/www/application/media # unzip ~/media-2019-10-16T15-11-00Z.zip
```

Comment:
I suggest we change all the commands to run from the application-folder. 

skip the CD command and: 
`/var/www/application # unzip ~/media-2019-10-16T15-11-00Z.zip -d media`
`/var/www/application # tree`